scripts/transform.sh

```diff
@@ -1,11 +1,34 @@
 #!/bin/bash
 
+# To run this script in development, first build the following packages:
+#     packages/@aws-cdk/assert
+#     packages/aws-cdk-lib
+#     tools/individual-pkg-gen
+
 set -euo pipefail
 scriptdir=$(cd $(dirname $0) && pwd)
 
 # Creates a symlink in each individual package's node_modules folder pointing
 # to the root folder's node_modules/.bin. This allows yarn to find the executables
 # it needs (e.g., jsii-rosetta) for the build.
+#
+# The reason yarn doesn't find the executables in the first place is that they are
+# not dependencies of each individual package -- nor should they be. They can't be
+# found in the lerna workspace, either, since it only includes the individual
+# packages. A possible solution that doesn't involve using this workaround function
+# is to add more packages to the workspace, which includes at least awslint and all
+# the tools/* packages, but potentially more, such as cloudformation-diff,
+# cloud-assembly-schema and others, which are dependencies of one or more of the
+# tools packages.
+#
+# In addition, we need to copy aws-cdk-lib inside individual-packages/ and remove
```

Comment:
The tenses here are a little weird.

Are you saying "A possible solution that doesn't involve using this workaround function is to [...]", and then "In addition,  we would need to copy aws-cdk-lib [...]", and then "Finally, we would need to restrict [...]"?

In that case, I would change the tenses, and _remove_ the two extra paragraphs (the new paragraphs to me seems to indicate you're tackling a new topic here, which I don't think is the case...?).

If that's not what you mean here, then I don't understand what these two last paragraphs are saying, which maybe means re-wording them a little bit is a good idea 😜.

Reply:
Ok, I can move them to a specific section, but they are all parts of the same thing; instead of implementing this stop-gap function, we need to try out a solution composed of three parts: 1) add more packages to the workspace, 2) copy aws-cdk-lib inside individual packages and 3) restrict the build only to the individual packages.

Reply:
I created a new GitHub issue, as we discussed yesterday and linked it from here.